app/util/links.ts

```diff
@@ -6,8 +6,102 @@
  * Copyright Oxide Computer Company
  */
 export const links: Record<string, string> = {
+  accessDocs: 'https://docs.oxide.computer/guides/configuring-access',
   cloudInitFormat: 'https://cloudinit.readthedocs.io/en/latest/explanation/format.html',
   cloudInitExamples: 'https://cloudinit.readthedocs.io/en/latest/reference/examples.html',
+  disksDocs: 'https://docs.oxide.computer/guides/managing-disks-and-snapshots',
+  firewallRulesDocs:
+    'https://docs.oxide.computer/guides/configuring-guest-networking#_firewall_rules',
   floatingIpsDocs: 'https://docs.oxide.computer/guides/managing-floating-ips',
-  ipPoolsDocs: 'https://docs.oxide.computer/guides/operator/ip-pool-management',
+  imagesDocs: 'https://docs.oxide.computer/guides/creating-and-sharing-images',
+  instancesDocs: 'https://docs.oxide.computer/guides/managing-instances',
+  keyConceptsIamPolicyDocs:
+    'https://docs.oxide.computer/guides/key-entities-and-concepts#iam-policy',
+  keyConceptsProjectsDocs:
+    'https://docs.oxide.computer/guides/key-entities-and-concepts#_projects',
+  projectsDocs: 'https://docs.oxide.computer/guides/onboarding-projects',
+  sledDocs:
+    'https://docs.oxide.computer/guides/architecture/service-processors#_server_sled',
+  snapshotsDocs:
+    'https://docs.oxide.computer/guides/managing-disks-and-snapshots#snapshots',
+  storageDocs:
+    'https://docs.oxide.computer/guides/architecture/os-hypervisor-storage#_physical_layer',
+  systemIpPoolsDocs: 'https://docs.oxide.computer/guides/operator/ip-pool-management',
+  systemMetricsDocs: 'https://docs.oxide.computer/guides/operator/system-metrics',
+  systemSiloDocs: 'https://docs.oxide.computer/guides/operator/silo-management',
+  vmsDocs:
+    'https://docs.oxide.computer/guides/deploying-workloads#_creating_virtual_machines',
+  vpcsDocs: 'https://docs.oxide.computer/guides/configuring-guest-networking',
+}
+
+// These are links (and associated titles) to help documentation
+export const docLinks = {
+  access: {
+    href: links.accessDocs,
+    linkText: 'Configuring Access and IAM',
+  },
+  disks: {
+    href: links.disksDocs,
+    linkText: 'Managing Disks',
+  },
+  firewallRules: {
+    href: links.firewallRulesDocs,
+    linkText: 'Firewall Rules',
+  },
+  floatingIps: {
+    href: links.floatingIpsDocs,
+    linkText: 'Floating IPs',
+  },
+  keyConceptsIam: {
+    href: links.keyConceptsIamPolicyDocs,
+    linkText: 'Key Concepts: Identity & Access Management Policy',
```

Comment:
This truncates. Maybe you could abbreviate IAM?

![Screenshot 2024-04-29 at 5 02 03 PM](https://github.com/oxidecomputer/console/assets/3612203/8124009e-8e08-4473-bfb7-b1ad84585d91)